lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -36,6 +36,12 @@ def execute(self) -> int:
     envvar="PBENCH_ACCESS_TOKEN",
     help="pbench server authentication token (will prompt if unspecified)",
 )
+@click.option(
+    "--access",
+    required=True,
```

Comment:
Same comment as for `move`?

Reply:
Consider using a [`click.Choice`](https://click.palletsprojects.com/en/8.1.x/options/#choice-options).